src/jit/gentree.h

```diff
@@ -2334,10 +2334,85 @@ struct GenTreeColon: public GenTreeOp
         {}
 };
 
-/* gtCall   -- method call      (GT_CALL) */
+// gtCall   -- method call      (GT_CALL)
 typedef class fgArgInfo *  fgArgInfoPtr;
 enum class InlineObservation;
 
+// Multi-reg return type descriptor of a GT_CALL node.
+// x64 Unix, Arm64, Arm32 and x86 allow a value to be returned in multiple
```

Comment:
As I mention above, I would prefer to make this a general ReturnTypeDesc - I don't see the value in going down this short-term path, when we really should have a general descriptor for all cases.